From 92dbf3f70eafd4974290fc488f51c05093950dbf Mon Sep 17 00:00:00 2001
From: Olivier Tilloy <olivier.tilloy@canonical.com>
Date: Mon, 12 Feb 2018 19:13:45 +0100
Subject: [PATCH] tdf#115659 Apparmor profile fixes for senddoc

Fix sending the current document by e-mail:
 - fix path to the installed senddoc executable
 - add missing exception for xdg-email
 - execute xdg-open and xdg-email unconfined

Change-Id: Ie7c5b112bd6c415036490e9f0f81063ca3ced5a1
Reviewed-on: https://gerrit.libreoffice.org/49614
Reviewed-by: Rene Engelhard <rene@debian.org>
Tested-by: Jenkins <ci@libreoffice.org>
---
 sysui/desktop/apparmor/program.senddoc | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

--- a/sysui/desktop/apparmor/program.senddoc
+++ b/sysui/desktop/apparmor/program.senddoc
@@ -12,7 +12,7 @@
 
 #include <tunables/global>
 
-profile libreoffice-senddoc INSTDIR-/usr/lib{,32,64}/libreoffice/program/senddoc {
+profile libreoffice-senddoc INSTDIR-program/senddoc {
   #include <abstractions/base>
 
   owner /tmp/lu**       rw,    #makes files like luRRRRR.tmp/lubRRRR.tmp where R is random
@@ -25,7 +25,8 @@ profile libreoffice-senddoc INSTDIR-/usr
   /usr/bin/basename     rmix,
   /bin/grep             rmix,
   /bin/uname            rmix,
-  /usr/bin/xdg-open     Cxr -> sanitized_helper,
+  /usr/bin/xdg-open     rPUx,
+  /usr/bin/xdg-email    rPUx,
   /dev/null             rw,
   INSTDIR-program/uri-encode rmpux,
   /usr/share/libreoffice/share/config/* r,
